packages/front-generator/.nycrc.yaml

```diff
@@ -13,7 +13,7 @@ exclude: [
   "src/generators/polymer2-typescript/**"
 ]
 
-statements: 90
-branches: 75
+statements: 89
+branches: 74
```

Comment:
We can decrease other coverage metrics to 80%, because 90% is could be too much. But I'm not agree to decrease branches values less than 75 (80 is goal). I spent a lot time to achieve such metrics for frontend-generator package.